### PR TITLE
Adding pkgdown using GitHub Actions.

### LINKED ISSUE #40: Create pkgdown site for easy consumption of manual w/out downloading
* **What are you trying to do?**

I would like to be able to have a website to point folks to for oysteR. 

* **What feature or behavior is this required for?**

A single URL that contains function documentation as well as user vignettes. 

* **How could we solve this issue? (Not knowing is okay!)**

Using pkgdown

* **Anything else?**

cc @bhamail / @DarthHater / @brittanybelle / @adrianpowell / @csgillespie

### PR DESCRIPTION
**Pull request by csgillespie**
* Also removing .travis
 * Closes #40 
 * As we are using gh-pages, and not doc, I've taken the `extra.css` file from #41. @JosiahParry contribution is acknowledged in the NEWS.md

cc @bhamail / @DarthHater / @brittanybelle / @adrianpowell / @csgillespie

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
   * Feature: Add `audit_description()` function
   * Feature: Add `expect_secure()` for the {testthat} package
   * Feature: Use `~/.ossindex/.oss-index-config`, if it exists
+  * Feature: Add a pkgdown website. Thanks to @josiahParry for the nice CSS.
   * Add Josiah Perry as contributor
```
Comment (JosiahParry) [inline NEWS.md:9]: Parry * ;)

Comment (csgillespie) [reply NEWS.md:9]: Sorry:(

Comment (bhamail) [reply NEWS.md:9]: Sorry couldn't resist:

https://www.youtube.com/watch?v=8cuihrjLNAo

Comment (JosiahParry) [reply NEWS.md]: loool! I sometimes say "Parry, like the verb." which isn't always understood.

**Comment by JosiahParry** [comment]
Exciting! Thanks!

**Pull request merged**